globals/modifiers.css

```diff
@@ -120,6 +122,16 @@
 .bgTextLight {background: var(--color-text-light);}
 .bgMuted {    background: var(--color-muted);}
 
+/* Color icons */
+.flPrimary {  fill: var(--color-primary);}
+.flSecondary {fill: var(--color-secondary);}
+.flError {    fill: var(--color-error);}
+.flAlert {    fill: var(--color-alert);}
+.flText {     fill: var(--color-text);}
+.flTextLight {fill: var(--color-text-light);}
+.flMuted {    fill: var(--color-muted);}
+.flWhite {    fill: var(--color-white);}
+
```

Comment:
Don't need these — the `fg` modifiers work on any SVGs that have `fill: currentColour` set on them (which is all `Icon`'s)